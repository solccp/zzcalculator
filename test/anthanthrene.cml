<?xml version="1.0"?>
<cml xmlns="http://www.xml-cml.org/schema">
  <molecule>
    <comment>SSS</comment>
    <atomArray>
      <atom id="a14" elementType="C" x3="      4.849984000000" y3="      0.000000000000" z3="      4.200207000000"/>
      <atom id="a20" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a19" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      3.500174000000"/>
      <atom id="a11" elementType="C" x3="      4.849983000000" y3="      0.000000000000" z3="      1.400069000000"/>
      <atom id="a12" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a13" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      3.500173000000"/>
      <atom id="a7" elementType="C" x3="      3.637487000000" y3="      0.000000000000" z3="      2.100104000000"/>
      <atom id="a8" elementType="C" x3="      3.637487000000" y3="      0.000000000000" z3="      3.500173000000"/>
      <atom id="a17" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      4.200208000000"/>
      <atom id="a18" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      1.400069000000"/>
    </atomArray>
    <bondArray>
      <bond atomRefs2="a14 a8" order="1"/>
      <bond atomRefs2="a14 a13" order="1"/>
      <bond atomRefs2="a20 a18" order="1"/>
      <bond atomRefs2="a20 a19" order="1"/>
      <bond atomRefs2="a19 a17" order="1"/>
      <bond atomRefs2="a19 a20" order="1"/>
      <bond atomRefs2="a11 a7" order="1"/>
      <bond atomRefs2="a11 a12" order="1"/>
      <bond atomRefs2="a12 a11" order="1"/>
      <bond atomRefs2="a12 a13" order="1"/>
      <bond atomRefs2="a12 a18" order="1"/>
      <bond atomRefs2="a13 a12" order="1"/>
      <bond atomRefs2="a13 a14" order="1"/>
      <bond atomRefs2="a13 a17" order="1"/>
      <bond atomRefs2="a7 a8" order="1"/>
      <bond atomRefs2="a7 a11" order="1"/>
      <bond atomRefs2="a8 a7" order="1"/>
      <bond atomRefs2="a8 a14" order="1"/>
      <bond atomRefs2="a17 a13" order="1"/>
      <bond atomRefs2="a17 a19" order="1"/>
      <bond atomRefs2="a18 a12" order="1"/>
      <bond atomRefs2="a18 a20" order="1"/>
    </bondArray>
  </molecule>
  <molecule>
    <comment>SSD</comment>
    <atomArray>
      <atom id="a20" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a19" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      3.500174000000"/>
      <atom id="a12" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a13" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      3.500173000000"/>
      <atom id="a17" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      4.200208000000"/>
      <atom id="a18" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      1.400069000000"/>
    </atomArray>
    <bondArray>
      <bond atomRefs2="a20 a18" order="1"/>
      <bond atomRefs2="a20 a19" order="1"/>
      <bond atomRefs2="a19 a17" order="1"/>
      <bond atomRefs2="a19 a20" order="1"/>
      <bond atomRefs2="a12 a13" order="1"/>
      <bond atomRefs2="a12 a18" order="1"/>
      <bond atomRefs2="a13 a12" order="1"/>
      <bond atomRefs2="a13 a17" order="1"/>
      <bond atomRefs2="a17 a13" order="1"/>
      <bond atomRefs2="a17 a19" order="1"/>
      <bond atomRefs2="a18 a12" order="1"/>
      <bond atomRefs2="a18 a20" order="1"/>
    </bondArray>
  </molecule>
  <molecule>
    <comment>SSR</comment>
    <atomArray>
      <atom id="a20" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a19" elementType="C" x3="      8.487467000000" y3="      0.000000000000" z3="      3.500174000000"/>
      <atom id="a13" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      3.500173000000"/>
      <atom id="a17" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      4.200208000000"/>
      <atom id="a18" elementType="C" x3="      7.274973000000" y3="      0.000000000000" z3="      1.400069000000"/>
      <atom id="a12" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      2.100103000000"/>
    </atomArray>
    <bondArray>
      <bond atomRefs2="a20 a18" order="1"/>
      <bond atomRefs2="a20 a19" order="1"/>
      <bond atomRefs2="a19 a17" order="1"/>
      <bond atomRefs2="a19 a20" order="1"/>
      <bond atomRefs2="a13 a12" order="1"/>
      <bond atomRefs2="a13 a17" order="1"/>
      <bond atomRefs2="a17 a13" order="1"/>
      <bond atomRefs2="a17 a19" order="1"/>
      <bond atomRefs2="a18 a12" order="1"/>
      <bond atomRefs2="a18 a20" order="1"/>
      <bond atomRefs2="a12 a13" order="1"/>
      <bond atomRefs2="a12 a18" order="1"/>
    </bondArray>
  </molecule>
  <molecule>
    <comment>SDR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>SRR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>DSR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>DDS</comment>
    <atomArray>
      <atom id="a14" elementType="C" x3="      4.849984000000" y3="      0.000000000000" z3="      4.200207000000"/>
      <atom id="a11" elementType="C" x3="      4.849983000000" y3="      0.000000000000" z3="      1.400069000000"/>
      <atom id="a7" elementType="C" x3="      3.637487000000" y3="      0.000000000000" z3="      2.100104000000"/>
      <atom id="a8" elementType="C" x3="      3.637487000000" y3="      0.000000000000" z3="      3.500173000000"/>
      <atom id="a12" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      2.100103000000"/>
      <atom id="a13" elementType="C" x3="      6.062478000000" y3="      0.000000000000" z3="      3.500173000000"/>
    </atomArray>
    <bondArray>
      <bond atomRefs2="a14 a8" order="1"/>
      <bond atomRefs2="a14 a13" order="1"/>
      <bond atomRefs2="a11 a7" order="1"/>
      <bond atomRefs2="a11 a12" order="1"/>
      <bond atomRefs2="a7 a8" order="1"/>
      <bond atomRefs2="a7 a11" order="1"/>
      <bond atomRefs2="a8 a7" order="1"/>
      <bond atomRefs2="a8 a14" order="1"/>
      <bond atomRefs2="a12 a11" order="1"/>
      <bond atomRefs2="a12 a13" order="1"/>
      <bond atomRefs2="a13 a12" order="1"/>
      <bond atomRefs2="a13 a14" order="1"/>
    </bondArray>
  </molecule>
  <molecule>
    <comment>DDD</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>DDR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>DRR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>RSS</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>RSD</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>RSR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>RDR</comment>
    <atomArray>
    </atomArray>
  </molecule>
  <molecule>
    <comment>RRR</comment>
    <atomArray>
    </atomArray>
  </molecule>
</cml>
