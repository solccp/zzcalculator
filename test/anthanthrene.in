   22
24
 C     0.000000     0.000000     0.000000
 C     0.000000     0.000000     1.400069
 C     1.212495     0.000000     2.100103
 C     2.424991     0.000000     1.400069
 C     2.424991     0.000000     0.000000
 C     1.212495     0.000000    -0.700034
 C     3.637487     0.000000     2.100104
 C     3.637487     0.000000     3.500173
 C     2.424992     0.000000     4.200207
 C     1.212496     0.000000     3.500173
 C     4.849983     0.000000     1.400069
 C     6.062478     0.000000     2.100103
 C     6.062478     0.000000     3.500173
 C     4.849984     0.000000     4.200207
 C     3.637485     0.000000    -0.700034
 C     4.849983     0.000000     0.000001
 C     7.274973     0.000000     4.200208
 C     7.274973     0.000000     1.400069
 C     8.487467     0.000000     3.500174
 C     8.487467     0.000000     2.100103
 C     7.274973     0.000000     0.000001
 C     6.062477     0.000000    -0.700033
