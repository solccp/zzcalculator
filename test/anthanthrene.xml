<zzpolynomial>
  <term>
    <order>0</order>
    <coefficient>10</coefficient>
  </term>
  <term>
    <order>1</order>
    <coefficient>12</coefficient>
  </term>
  <term>
    <order>2</order>
    <coefficient>3</coefficient>
  </term>
  <total>25</total>
</zzpolynomial>
