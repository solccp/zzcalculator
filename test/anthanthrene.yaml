---
structure_id: 1
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 15]
- [5, 4]
- [14, 13]
- [17, 19]
- [20, 18]
- [12, 11]
- [7, 8]
---
structure_id: 2
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 15]
- [5, 4]
- [14, 8]
- [7, 11]
- [20, 19]
- [18, 12]
- [17, 13]
---
structure_id: 3
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 15]
- [5, 4]
- [14, 8]
- [7, 11]
- [20, 18]
- [19, 17]
- [13, 12]
---
structure_id: 4
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 15]
- [5, 4]
- [14, 8]
- [7, 11]
rings:
- [20, 18, 12, 13, 17, 19]
---
structure_id: 5
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 15]
- [5, 4]
- [17, 19]
- [20, 18]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 6
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 11]
- [15, 5]
- [4, 7]
- [8, 14]
- [20, 19]
- [18, 12]
- [13, 17]
---
structure_id: 7
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 11]
- [15, 5]
- [4, 7]
- [8, 14]
- [20, 18]
- [19, 17]
- [13, 12]
---
structure_id: 8
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [16, 11]
- [15, 5]
- [4, 7]
- [8, 14]
rings:
- [20, 18, 12, 13, 17, 19]
---
structure_id: 9
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [8, 14]
- [20, 19]
- [18, 12]
- [13, 17]
rings:
- [16, 11, 7, 4, 5, 15]
---
structure_id: 10
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [8, 14]
- [20, 18]
- [19, 17]
- [12, 13]
rings:
- [16, 11, 7, 4, 5, 15]
---
structure_id: 11
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 21]
- [8, 14]
rings:
- [16, 11, 7, 4, 5, 15]
- [20, 18, 12, 13, 17, 19]
---
structure_id: 12
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [22, 16]
- [21, 18]
- [20, 19]
- [15, 5]
- [4, 7]
- [17, 13]
- [11, 12]
- [14, 8]
---
structure_id: 13
number_of_atoms: 22
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 19]
- [15, 5]
- [4, 7]
- [8, 14]
- [13, 17]
rings:
- [22, 16, 11, 12, 18, 21]
---
structure_id: 14
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 10]
- [4, 7]
- [9, 8]
- [14, 13]
- [17, 19]
- [20, 18]
- [12, 11]
---
structure_id: 15
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 4]
- [10, 9]
- [17, 19]
- [20, 18]
- [14, 13]
- [12, 11]
- [8, 7]
---
structure_id: 16
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 4]
- [10, 9]
- [17, 19]
- [20, 18]
- [14, 8]
- [7, 11]
- [12, 13]
---
structure_id: 17
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 4]
- [10, 9]
- [17, 19]
- [20, 18]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 18
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 4]
- [10, 9]
- [17, 13]
- [19, 20]
- [18, 12]
- [11, 7]
- [14, 8]
---
structure_id: 19
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [3, 4]
- [10, 9]
- [11, 7]
- [14, 8]
rings:
- [17, 13, 12, 18, 20, 19]
---
structure_id: 20
number_of_atoms: 22
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [22, 21]
- [11, 12]
- [18, 20]
- [14, 13]
- [17, 19]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 21
number_of_atoms: 22
double_bonds:
- [10, 9]
- [15, 16]
- [22, 21]
- [7, 11]
- [8, 14]
- [19, 20]
- [17, 13]
- [12, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 22
number_of_atoms: 22
double_bonds:
- [10, 9]
- [15, 16]
- [22, 21]
- [7, 11]
- [8, 14]
- [19, 17]
- [13, 12]
- [20, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 23
number_of_atoms: 22
double_bonds:
- [10, 9]
- [15, 16]
- [22, 21]
- [7, 11]
- [8, 14]
rings:
- [1, 2, 3, 4, 5, 6]
- [19, 17, 13, 12, 18, 20]
---
structure_id: 24
number_of_atoms: 22
double_bonds:
- [10, 9]
- [15, 16]
- [22, 21]
- [7, 8]
- [11, 12]
- [18, 20]
- [14, 13]
- [17, 19]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 25
number_of_atoms: 22
double_bonds:
- [10, 9]
- [15, 16]
- [22, 21]
- [17, 19]
- [20, 18]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
