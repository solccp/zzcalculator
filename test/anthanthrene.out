 10 + 12 x + 3 x^2
 total: 25
